rmc-docs/src/tool-comparison.md

```diff
@@ -0,0 +1,20 @@
+# Comparison with other tools
+
+**Fuzzing** (for example, with [`cargo-fuzz`](https://github.com/rust-fuzz/cargo-fuzz)) is a blind approach to random testing.
```

Comment:
Could we avoid the word "blind" here? "General", "broad approach to automating random testing"?